### PR TITLE
QA + CS fixes

### PR DESCRIPTION
**Pull request by michalbundyra**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

CS fixes and QA improvements (import internal functions and constants!)

### CODE REVIEW

File: src/Element/Captcha.php [hunk 1]
```diff
@@ -86,7 +92,7 @@ public function getInputSpecification()
             'name' => $this->getName(),
             'required' => true,
             'filters' => [
-                ['name' => 'Laminas\Filter\StringTrim'],
+                ['name' => \Laminas\Filter\StringTrim::class],
```
Comment (froschdesign) [inline src/Element/Captcha.php]: No import here?

File: test/TestAsset/AddressFieldset.php [hunk 2]
```diff
@@ -37,7 +38,7 @@ class_exists(ClassMethodsHydrator::class)
         $this->add($street);
         $this->add($city);
 
-        $phones = new \Laminas\Form\Element\Collection('phones');
+        $phones = new Element('phones');
```
Comment (froschdesign) [inline test/TestAsset/AddressFieldset.php]: This is the wrong class!

**Comment by froschdesign** [comment]
🚢

**Comment by michalbundyra** [comment]
@froschdesign Thanks a lot for checking this huge PR!

**Comment by froschdesign** [comment]
I checked the changes locally, even the last ones. Looks good! 👍

**Pull request merged**